libs/eo/landing-page/shell/tsconfig.lib.json

```
@@ -8,5 +8,5 @@
     "types": []
   },
   "exclude": ["src/test-setup.ts", "**/*.spec.ts", "**/*.test.ts"],
-  "include": ["**/*.ts"]
+  "include": ["**/*.ts", "../routing/src/lib/eo-landing-page-relative-url.ts"]
```

issue: This must have been done by VS Code when you moved the file. Remove this setting.
```suggestion
  "include": ["**/*.ts"]
```